content/_data/indexpage/carousel.yml

```
@@ -1,3 +1,13 @@
+# 2022 Elections
+- :title: Jenkins Elections 2022
+  :intro: Voting for the Jenkins Board and Officer Elections 2022 begin this Thursday, Nov 17.
+    Voter registration is happening now until Nov 17
+  :image:
+    :src: images/logos/needs-you/Jenkins_Needs_You-transparent.png
+    :height: 320px
+  :call_to_action:
+    :text: More info
+    :href: https://www.jenkins.io/blog/2022/10/20/jenkins-election-announcement/
```

```suggestion
    :href: blog/2022/10/20/jenkins-election-announcement/
```
Something like that? I've looked [at previous jumbotron entries](https://github.com/jenkins-infra/jenkins.io/pull/4209) but they were using a previous syntax like `slides << {:href => expand_link('blog/2021/03/19/SheCodeAfrica-announcement/'),` which doesn't seems to be used anymore.